src/Files/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -21,6 +21,11 @@
     SecondaryButtonText="{x:Bind ViewModel.SecondaryButtonText, Mode=OneWay}"
     Style="{StaticResource DefaultContentDialogStyle}"
     mc:Ignorable="d">
+    <i:Interaction.Behaviors>
+        <icore:EventTriggerBehavior EventName="Closing">
+            <icore:InvokeCommandAction Command="{x:Bind ViewModel.ClosingCommand, Mode=OneWay}" />
```

Comment:
Won't trigger the command because command args don't match (?)
Either way, I prefer to have the event handler in the codebehind calling a command in the ViewModel

Reply:
Like in fc5f172190fd557c3541eb0477186bb4aa96d9b9?

Reply:
The reason is to prevent calling a method when it can't be executed.
![image](https://user-images.githubusercontent.com/53011783/152642181-1d6ec43b-18f8-48f4-880c-69285c596be5.png)
(CommunityToolkit.Mvvm.Input.RelayCommand<T>)